### PR TITLE
Documentation for strong Gröbner bases over the integers

### PR DESCRIPTION
**Pull request by ederc**

### CODE REVIEW

**Comment by thofma** [comment]
Do you think it would make sense to add a reference?

**Comment by wdecker** [comment]
How about Adams/Loustaunau as a text book reference?

**Comment by ederc** [comment]
I've added the Adams/Loustaunau reference, take a look.

**Comment by wdecker** [comment]
Thanks!

File: docs/src/CommutativeAlgebra/ideals.md [hunk 1]
```diff
@@ -214,9 +214,24 @@ leading_ideal(I::MPolyIdeal)
...
+Gröbner bases and normal forms plays an important role. This is done when 
+computing strong Gröbner bases which ensure the following property: 
+For any element of an ideal its leading term is divisible by a leading term of an 
+element of a corresponding strong Gröbner basis.
 
+The textbook
 
+- [AL94](@cite)
+
+provides details on theory and algorithms as well as references.
```
Comment (fingolfin) [inline docs/src/CommutativeAlgebra/ideals.md]: Hmm, why put this into a list, though?
```suggestion
The textbook [AL94](@cite) provides details on theory and algorithms as well as references.
```

Anyway, this is a matter of taste, I guess *shrug*

**Review by fingolfin — APPROVED** [review]

**Pull request merged**